lib/smart_answer_flows/coronavirus-business-support/outcomes/results.govspeak.erb

```diff
@@ -122,4 +122,12 @@
     [Find out more](https://www.google.com)
     $CTA
   <% end %>
+
+  <% if calculator.show?(:large_business_loan_scheme) %>
+    $CTA
+    **[Coronavirus Large Business Interruption Loan Scheme](https://www.gov.uk/guidance/apply-for-the-coronavirus-large-business-interruption-loan-scheme)**
```

Comment:
Ah do we have a preference for using absolute URLs? I'll update for consistency

Reply:
Sorry no preference that was just me copy and pasting. Should probably use relative, but that won't work in the Heroku app.